EvolvSwiftSDK/EvolvSwiftSDK/Sources/Data Model/Configuration.swift

```diff
@@ -115,12 +115,24 @@ public struct ExperimentKey: Decodable, Equatable {
         values = try? container.decode(Bool.self, forKey: .init(stringValue: "_values"))
         initializers = try container.decode(Bool.self, forKey: .init(stringValue: "_initializers"))
         
+        let keyPath = container.codingPath
+            .dropFirst(2)
```

Comment:
What are we dropping here?

Reply:
The coding path of json that led to the current object node.
Without the drop it's something like this: ["", "_experiments", "home", "background"]